transport_nantes/mailing_list/templates/mailing_list/mailing_list_user_status.html

```diff
@@ -0,0 +1,14 @@
+{% extends 'asso_tn/base_mobilitain.html' %}
+
+{% block content %}
+<div class="container-fluid p-3">
+    <div class="row">
+        <div class="col-sm-12 col-md-10 mx-auto">
+            {% comment %} Create a list with mailing_list {% endcomment %}
+            {% include "mailing_list/status/list_mp.html"  with div_title="Mes newsletters" base_list=mailing_lists%}
+            {% comment %} Create a list with petition {% endcomment %}
```

Comment:
![image](https://user-images.githubusercontent.com/70256364/156369844-63147c51-9e38-45d0-8b04-3158d8d8d491.png)

Mes Petition is still present,  we discussed and set on petitions aren't actionable for users, so we don't need to include them. 
beside, the mailing_list/mailing_list_user_status.html is still pushed to the very left, could you give a shot to a centered version please ? :slightly_smiling_face: